x-pack/metricbeat/module/statsd/_meta/docs.asciidoc

```diff
@@ -1,2 +1,79 @@
-The Statsd module is a Metricbeat module which opens a UDP port and listens for statsd metrics.
+[role="xpack"]
+== Statsd module
 
+The `statsd` module is a Metricbeat module which spawns a UDP server and listens for metrics in StatsD compatible
+format.
+
+[float]
+=== Metric types
+
+The module supports the following types of metrics:
+
+*Counter (c)*:: Measurement which accumulates over period of time until flushed (value set to 0).
+
+*Gauge (g)*:: Measurement which can increase, decrease or be set to a value.
+
+*Timer (ms)*:: Time measurement (in milliseconds) of an event.
+
+*Histogram (h)*:: Time measurement, alias for timer.
+
+*Set (s)*:: Measurement which counts unique occurrences until flushed (value set to 0).
+
+[float]
+=== Configuration options
+
+This module has some configuration options for controlling its behavior. The
+following example shows all configuration options.
+
+[source,yaml]
+----
+- module: statsd
+  metricsets: ["server"]
+  host: "localhost"
+  port: "8125"
+  #ttl: "30s"
+----
+
+This module also supports the
+<<module-standard-options-{modulename},standard configuration options>>
+described later.
+
+*`ttl`*:: It defines how long a metric will be reported after it was last recorded.
+Irrespective of the given ttl, metrics will be reported at least once.
+A ttl of zero means metrics will never expire.
+
+include::{docdir}/metricbeat-options.asciidoc[]
+
+[float]
+=== Metricsets
+
+Currently, there is only `server` metricset in `statsd` module.
+
+[float]
+==== `server`
+The metricset collects metric data sent using UDP and publishes them under the `statsd` prefix.
+
+[float]
+=== Development
```

Comment:
I think development notes should be in a `README.md` instead.

Reply:
Fixed.